lib/assets/connection_cell/main.js

```diff
@@ -259,13 +260,13 @@ export function init(ctx, info) {
       <label class="hidden-checkbox">
         <input
           type="checkbox"
-          :value="fields.password_secret"
-          name="password_secret"
-          v-model="fields.password_secret"
-          true-value="true" false-value=""
+          :value="fields.use_password_secret"
+          name="use_password_secret"
+          v-model="fields.use_password_secret"
+          true-value="true" false-value="false"
           class="hidden-checkbox-input"
         />
-        <span v-if="fields.password_secret" class="button--sm">
+        <span v-if="fields.use_password_secret === 'true'" class="button--sm">
```

Comment:
Can we make it an actual boolean? I think his will simplify things :)

Reply:
I tried that but couldn't succeed, as we need to force the checkbox to send a value. An option is to roll the `false-value` back to an empty string and use a conditional in attrs. To be honest, I prefer to keep the attrs simpler and leave this little mess on JS's side. Wdyt?

Reply:
By default Vue `v-model` works with booleans, so we should be good on this part. As for sending the value I assume you mean `handleFieldChange`, where we operate on the DOM element directly. One way would be to checking the input type and for checkbox we look at `input.checked` rather than `input.value`, **but** I think we can actually just do `this.fields[name]` to get the new value in all cases :)